javaslang/src/test/java/javaslang/collection/VectorPropertyTest.java

```diff
@@ -0,0 +1,260 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2016 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.collection;
+
+import javaslang.Function2;
+import javaslang.Tuple;
+import javaslang.Tuple2;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.function.Function;
+
+import static javaslang.Function2.constant;
+import static javaslang.collection.BitMappedTrie.BRANCHING_BASE;
+import static javaslang.collection.BitMappedTrie.branchingFactor;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class VectorPropertyTest {
+    @Before
+    public void setUp() { BRANCHING_BASE = 2; }
```

Comment:
It seems risky to me to not change this back to its original value. This can effect other tests run in the same JVM. For example, running `VectorPropertyTest` and `VectorTest` subsequently will mean that `VectorTest` runs with a branching base of `2`, and branching factor of `4`. However, running `VectorTest` individually will use the default values, such as base `5` and factor `32`.

Therefore, I'd suggest adding an `@After` method with setting `BRANCHING_BASE` back to `5`.